tests/hypervisor/test_multi_hypervisors.py

```diff
@@ -26,13 +30,82 @@ def test_multi_hypervisors_report_together(self, virtwho):
         :customerscenario: false
         :upstream: no
         :steps:
-            1. Create the virt-who config files for the multi hypervisors list
-            2. Run the virt-who service
+            1. Create the separated virt-who config file for each hypervisor
+            2. Run the virt-who service and check the register server
+            3. Combine all the hypervisor config files to one
+            4. Run the virt-who service and check the register server
 
         :expectedresults:
             1. Succeed to run the virt-who, no error messages in the rhsm.log
+            2. All the hypervisors could be found in the register server
         """
-        for mode in hypervisors_list():
-            hypervisor_create(mode)
-        result = virtwho.run_service()
-        assert result["error"] == 0 and result["send"] == 1 and result["thread"] == 1
+        file_types = ["separated", "single"]
+        single_file = "/etc/virt-who.d/virtwho_multi.conf"
+        try:
+            for typ in file_types:
```

Comment:
Since the `type` is the Python function, update the value to `file_type` would be better?
```suggestion
            for file_type in file_types:
```

Reply:
done, thanks.